component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/TOTPAuthenticator.java

```diff
@@ -732,6 +732,8 @@ private void handleTotpVerificationFail(AuthenticationContext context) throws Au
             updatedClaims.put(TOTPAuthenticatorConstants.ACCOUNT_UNLOCK_TIME_CLAIM, String.valueOf(unlockTime));
             updatedClaims.put(TOTPAuthenticatorConstants.FAILED_LOGIN_LOCKOUT_COUNT_CLAIM,
                     String.valueOf(failedLoginLockoutCountValue + 1));
+			updatedClaims.put(TOTPAuthenticatorConstants.ACCOUNT_LOCKED_REASON_CLAIM_URI,
+					"MAX_TOTP_ATTEMPTS_EXCEEDED");
```

Comment:
Can we define a constant for this value? Also I think we have a formatting error.

Reply:
Add the constant with 42e3e1a0fef90f8e9eeb67632ef0a7eeff5132d2. Seems like this formatting error occurs due to an issue in GitHub. This is not shown like this in the local.

![Screenshot 2022-05-12 at 10 52 48 AM](https://user-images.githubusercontent.com/32576163/167997913-933ccd29-7e82-4b96-82f4-91ee7c1d2306.png)